app/src/main/java/org/simple/clinic/contactpatient/views/CallPatientView_Old.kt

```diff
@@ -0,0 +1,104 @@
+package org.simple.clinic.contactpatient.views
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.view.View
+import android.widget.Toast
+import androidx.constraintlayout.widget.ConstraintLayout
+import org.simple.clinic.R
+import org.simple.clinic.databinding.ContactpatientCallpatientOldBinding
+import org.simple.clinic.patient.Gender
+import org.simple.clinic.patient.displayLetterRes
+
+private typealias AgreedToVisitClicked_Old = () -> Unit
+private typealias RemindToCallLaterClicked_Old = () -> Unit
+private typealias RemoveFromOverdueListClicked_Old = () -> Unit
+private typealias NormalCallButtonClicked_Old = () -> Unit
+private typealias SecureCallButtonClicked_Old = () -> Unit
+
+class CallPatientView_Old(
+    context: Context,
+    attributeSet: AttributeSet
+) : ConstraintLayout(context, attributeSet) {
+
+  private var binding: ContactpatientCallpatientOldBinding? = null
+
+  private val callResultsGroup
+    get() = binding!!.callResultsGroup
+
+  private val secureCallingGroup
+    get() = binding!!.secureCallingGroup
+
+  private val agreedToVisitTextView
+    get() = binding!!.agreedToVisitTextView
+
+  private val remindToCallLaterTextView
+    get() = binding!!.remindToCallLaterTextView
+
+  private val removeFromOverdueListTextView
+    get() = binding!!.removeFromOverdueListTextView
+
+  private val normalCallButton
+    get() = binding!!.normalCallButton
+
+  private val secureCallButton
+    get() = binding!!.secureCallButton
+
+  private val nameTextView
+    get() = binding!!.nameTextView
+
+  private val phoneNumberTextView
+    get() = binding!!.phoneNumberTextView
+
+  var callResultSectionVisible: Boolean = false
+    set(value) {
+      field = value
+      callResultsGroup.visibility = if (field) View.VISIBLE else View.GONE
+    }
+
+  var secureCallingSectionVisible: Boolean = false
+    set(value) {
+      field = value
+      secureCallingGroup.visibility = if (field) View.VISIBLE else View.GONE
+    }
+
+  var overdueListChangesEnabled: Boolean = false
+  set(value) {
+    field = value
+  }
+
+  var agreedToVisitClickedOld: AgreedToVisitClicked_Old? = null
+
+  var remindToCallLaterClickedOld: RemindToCallLaterClicked_Old? = null
+
+  var removeFromOverdueListClickedOld: RemoveFromOverdueListClicked_Old? = { Toast.makeText(context, "WIP", Toast.LENGTH_SHORT).show() }
+
+  var normalCallButtonClickedOld: NormalCallButtonClicked_Old? = null
+
+  var secureCallButtonClickedOld: SecureCallButtonClicked_Old? = null
```

Comment:
We don't need to rename them as `old`, because they are tied to the instance of this view.

Reply:
These are typealiases and I was getting a redeclaration error when I was using the orginal name

Reply:
Also, I renamed the variables as this is being called from `ContactPatientBottomSheet` so for consistency

Reply:
Yeah, what I meant was the variable names.